sdk/core/azure-core/azure/core/credentials.py

```diff
@@ -24,7 +24,12 @@ class TokenCredential(Protocol):
     """Protocol for classes able to provide OAuth tokens."""
 
     def get_token(
-        self, *scopes: str, claims: Optional[str] = None, tenant_id: Optional[str] = None, **kwargs: Any
+        self,
+        *scopes: str,
+        claims: Optional[str] = None,
```

Comment:
for kwargs

```suggestion
        *, 
        claims: Optional[str] = None,
```

Reply:
We already have `*scopes: str,` I don't think we need `*,` again?

Reply:
hmm I figured that scope is a param and we are taking it as an unpacked tuple ( basically same as `*args` ). It wasn't complaining about that?

sounds like a case for @kristapratico